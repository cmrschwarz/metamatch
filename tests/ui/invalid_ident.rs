use metamatch::unquote;

fn main() {
    unquote! {
       ident("3")
    }
}
