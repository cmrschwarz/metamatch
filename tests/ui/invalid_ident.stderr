error: cast failed: invalid identifier `3`
 --> tests/ui/invalid_ident.rs:5:8
  |
5 |        ident("3")
  |        ^^^^^
